web/templates/draft.hbs

```diff
@@ -16,7 +16,7 @@
                     </div>
                 </span>
                 <span class="recipient_bar_controls"></span>
-                <div class="recipient_row_date" title="{{t 'Last modified'}}">{{ time_stamp }}</div>
+                <div class="recipient_row_date tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Last modified'}}">{{ time_stamp }}</div>
```

Comment:
Let's show the detailed time like we do in the message feed.
Eg., `Last modified on 2/6/2024 at 11:02:14 PM (UTC +05:30)`